src/components/store/modals/ModalRegisterDapp.vue

```diff
@@ -37,26 +37,30 @@
           label="Name"
           type="text"
           required
+          maxlength="200"
           :validationMessage="validationErrors['name']"
         />
         <Input
```

Comment:
How about adding another input section for selecting the sender's account, like the dropdown option in the balance transfer.